openassessment/xblock/static/js/src/studio/oa_edit_listeners.js

```diff
@@ -174,7 +174,7 @@ OpenAssessment.StudentTrainingListener.prototype = {
         if (changed) {
             this.displayAlertMsg(
                 gettext("Option Deleted"),
-                gettext("You've deleted all the options for this criterion. The system has removed the criterion from the sample responses in the Student Training step.")
+                gettext("You've deleted all the options for this criterion. The system has removed the criterion from the sample responses in the Learner Training step.")
```

Comment:
"You've deleted..." -> "You have deleted..."
"The criterion has been removed from the example responses in the Learner Training step." 
(note change from "sample" to "example")